sched/environ/environ.h

```diff
@@ -76,6 +78,46 @@ extern "C"
 
 int env_dup(FAR struct task_group_s *group);
 
+/****************************************************************************
+ * Name: env_set
+ *
+ * Description:
+ *   Set the internal environment structure of a task.  This is the action
+ *   that is performed when a new task is created: The new task has a
+ *   private, exact duplicate of the parent task's environment.
+ *
+ * Input Parameters:
+ *   group - The child task group to receive the newly allocated copy of the
+ *           parent task groups environment structure.
+ *   envp  - Pointer to the environment strings.
+ *
+ * Returned Value:
+ *   zero on success
+ *
+ * Assumptions:
+ *   Not called from an interrupt handler.
+ *
+ ****************************************************************************/
+
+int env_set(FAR struct task_group_s *group, FAR char * const *envp);
```

Comment:
it's more simple to merge env_dup and env_set into one and handle envp == NULL as get the environ from parent.

Reply:
This is exactly how I did it in the first version of this PR so no problem

Reply:
Done